cns/requestcontroller/kubecontroller/crdrequestcontroller.go

```diff
@@ -227,20 +230,20 @@ func (crdRC *crdRequestController) initCNS() error {
 	}
 
 	// Call cnsclient init cns passing those two things
-	return crdRC.CNSClient.InitCNSState(ncRequest, podInfoByIP)
+	return crdRC.CNSClient.InitCNSState(ncRequestReference, podInfoByIP)
```

Comment:
pass &ncRequest only

Reply:
Here's the thing. If the crd isn't found, we want to pass a nil pointer right? That's why I added this ncRequestReference variable. ncRequestReference is nil if the crd wasn't found.

Reply:
Nvm, saw your other comment, making a pointer